### DISCUSSION TITLE
會提供 TGE 測試鏈合約以供試驗嗎?

### DISCUSSION THREAD

**Discussion by gasolin** [lun-dao/LunDAO#54] [General]
是不是要提供使用說明 (或可供操作的網頁)

測試鏈可以加上 faucet https://fauceth.komputing.org/

**Comment by yurenju** [comment]
我沒有很懂耶，是說哪部分會需要 faucet 呢？

**Comment by hydai** [comment]
這個問題有三個地方我不太了解：
1. 合約是指哪一個部分的合約？
2. 試驗是包含哪些行為需要測試？
3. faucet 的部分又是為了哪個部分提供的？

**Comment by gasolin** [comment]
有意願參加 TGE 的人也許會有以下考量

1. 沒試過 Arbitrum 網路 (要怎麼跨鏈, 需要使用說明)
2. TGE 合約是否能運作 (驗證存 ETH 得 LUN 合約)
3. 在測試網的話怎麼拿到測試 ETH (需要從 faucet 取得測試 ETH)
4. 不會直接和合約交互 (需可供操作的網頁介面)

**Reply by yurenju** [reply]
TGE 結束啦 ＸＤＤＤ

**Reply by hydai** [reply]
哈哈哈，對，難怪我一直想不懂發生什麼事了XD

**Reply by yurenju** [reply]
可能我寫得不夠清楚，慚愧...

不過因為 TGE 結束的區塊是 [13975700][1] 是週一的早上，所以已經結束了。接下來就會是把 LUN 搬到 Arbitrum 去

[1]: https://etherscan.io/block/13975700

**Reply by gasolin** [reply]
LOL
